src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -70,16 +71,26 @@ public function evaluate(array $functions, array $values)
             case self::PROPERTY_CALL:
                 $obj = $this->nodes['node']->evaluate($functions, $values);
                 if (!\is_object($obj)) {
+                    if (is_null($obj) && $this->nodes['attribute']->isNullSafe) {
```

Comment:
`if (null === $obj [...]` + I'd suggest to move this "if" outside of the parent "if"